Windows build failed:
```
Error: ..\src\sources\soundsourcemediafoundation.cpp(532): error C2870: '<Unknown>': a namespace definition must appear either at file scope or immediately within another namespace definition
Error: ..\src\sources\soundsourcemediafoundation.cpp(537): error C2601: 'mixxx::`anonymous-namespace'::configureMediaType': local function definitions are illegal
..\src\sources\soundsourcemediafoundation.cpp(532): note: this line contains a '{' which has not yet been matched
Error: ..\src\sources\soundsourcemediafoundation.cpp(695): error C2601: 'mixxx::SoundSourceMediaFoundation::configureAudioStream': local function definitions are illegal
..\src\sources\soundsourcemediafoundation.cpp(276): note: this line contains a '{' which has not yet been matched
Error: ..\src\sources\soundsourcemediafoundation.cpp(791): error C2601: 'mixxx::SoundSourceMediaFoundation::readProperties': local function definitions are illegal
..\src\sources\soundsourcemediafoundation.cpp(276): note: this line contains a '{' which has not yet been matched
..\src\sources\soundsourcemediafoundation.cpp(813): fatal error C1506: unrecoverable block scoping error
[533/818] Building CXX object CMakeFiles\mixxx-lib.dir\src\controllers\hid\hidcontroller.cpp.obj
ninja: build stopped: subcommand failed.
Error: Process completed with exit code 1.
```

--- a/src/sources/soundsourcemediafoundation.cpp
+++ b/src/sources/soundsourcemediafoundation.cpp
@@ -370,30 +370,31 @@ ReadableSampleFrames SoundSourceMediaFoundation::readSampleFramesClamped(
         }
         DEBUG_ASSERT(pSample != nullptr);
         SINT readerFrameIndex = m_streamUnitConverter.toFrameIndex(streamPos);
+        // TODO: Fix debug assertion in else arm. It has been commented
+        // out deliberately to prevent crashes in debug builds.
+        // https://bugs.launchpad.net/mixxx/+bug/1899242
         if (m_currentFrameIndex == kUnknownFrameIndex) {
             // Unknown position after seeking
             m_currentFrameIndex = readerFrameIndex;
-            /*
-            kLogger.debug()
-                    << "Stream position (in sample frames) after seeking:"
-                    << "target =" << writableSampleFrames.frameIndexRange().end()
-                    << "current =" << readerFrameIndex;
-            */
-        } else {
-            // Both positions should match, otherwise readerFrameIndex
-            // is inaccurate due to rounding errors after conversion from
-            // stream units to frames! But if this ever happens we better
-            // trust m_currentFrameIndex that is continuously updated while
-            // reading in forward direction.
-            VERIFY_OR_DEBUG_ASSERT(m_currentFrameIndex == readerFrameIndex) {
-                kLogger.debug()
-                        << "streamPos [100 ns] =" << streamPos
-                        << ", sampleRate =" << getSignalInfo().getSampleRate();
-                kLogger.warning()
-                        << "Stream position (in sample frames) while reading is inaccurate:"
-                        << "expected =" << m_currentFrameIndex
-                        << "actual =" << readerFrameIndex;
-            }
+            //     kLogger.debug()
+            //             << "Stream position (in sample frames) after seeking:"
+            //             << "target =" << writableSampleFrames.frameIndexRange().end()
+            //             << "current =" << readerFrameIndex;
+            // } else {
+            //     // Both positions should match, otherwise readerFrameIndex
+            //     // is inaccurate due to rounding errors after conversion from
+            //     // stream units to frames! But if this ever happens we better
+            //     // trust m_currentFrameIndex that is continuously updated while
+            //     // reading in forward direction.
+            //     VERIFY_OR_DEBUG_ASSERT(m_currentFrameIndex == readerFrameIndex) {
+            //         kLogger.debug()
+            //                 << "streamPos [100 ns] =" << streamPos
+            //                 << ", sampleRate =" << getSignalInfo().getSampleRate();
+            //         kLogger.warning()
+            //                 << "Stream position (in sample frames) while reading is inaccurate:"
+            //                 << "expected =" << m_currentFrameIndex
+            //                 << "actual =" << readerFrameIndex;
+            //     }
         }
 
         DWORD dwSampleBufferCount = 0;

